package-lock.json

```diff
@@ -0,0 +1,5024 @@
+{
```

Comment:
Are you sure this file should be included?  I think you should remove it from git.